sdk/trace/trace_test.go

```diff
@@ -1626,12 +1626,15 @@ func TestCustomClock(t *testing.T) {
 	now := time.Now()
 	tp := NewTracerProvider(WithSyncer(te), WithClock(newFrozenClock(now)))
 	tracer := tp.Tracer("custom-clock")
+
 	_, span := tracer.Start(context.Background(), "test-frozen-clock")
 	time.Sleep(time.Microsecond * 2)
 	span.End()
+	assert.Equal(t, len(te.Spans()), 1, "Should only have one span")
```

Comment:
```suggestion
	require.Len(t, te.Spans(), 1, "should only have one span")
```

1. it has to be `require`. otherwise, the test will panic later
2. there is a `Len` convenience function 😉

Reply:
Done. BTW there's a race test fails sometimes in the CI, I assume its not relevant to this PR?

Reply:
> BTW there's a race test fails sometimes in the CI, I assume its not relevant to this PR?

Yes. It is not relevant to this PR.

Of course, if you want (and have time) I encourage you to take a look and try solving it 😉